### ISSUE TITLE
fast run and parcool not starting until you click alt p

### ISSUE THREAD

**Issue by VaporeonScripts** [alRex-U/ParCool#302]
**Summary**
fast run and any of parcool stuff wont work until u click alt p, which is annoying for modpack users, mind fixing it?

**Running Environment**
+ Minecraft Version: 1.21.1
+ ParCool Version: latest neoforge
+ Forge Version (optional): neoforge .69

**Detail**
Detail about the bug
fast run and any of parcool stuff wont work until u click alt p, like if it never existed
**Additional Data**
no need?

**Comment by alRex-U** [comment]
This may be caused by synchronization of limitation.
Does this occur even if only ParCool is installed? and always occur?

**Comment by VaporeonScripts** [comment]
Yeah,also ur config doesn't save when u change stuff in game,pls fix ur mod,and make h wall run auto mode better xd
